src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin on-all {
+  @media (min-width: 0) {
+    grid-template-columns: 1, 1fr;
```

Comment:
There is a syntax error in 'grid-template-columns: 1, 1fr;'. The correct syntax should be 'grid-template-columns: repeat(1, 1fr);' to ensure proper grid layout.